app/src/androidTest/java/com/github/h3lp3rs/h3lp/ChatUiTest.kt

```diff
@@ -32,28 +42,67 @@ class ChatUiTest {
     private lateinit var conversationFrom: Conversation
     private lateinit var conversationTo: Conversation
 
+    private lateinit var db: Database
```

Comment:
I'm not sure why this is defined as a lateinit variable, indeed, you can simply access db using databaseOf(MESSAGES) once you have established the mocked version in the test setup function. This is to be more consistent with other test classes where it is done this way.